library/src/main/scala/scala/scalajs/js/PropertyDescriptor.scala

```diff
@@ -10,12 +10,21 @@
 
 package scala.scalajs.js
 
-@native
+import scala.scalajs.js
```

Comment:
I can add an additional commit with that change everywhere. It kind of fits the theme of the PR.